newIDE/app/src/UI/Toggle.js

```diff
@@ -17,10 +17,11 @@ type Props = {|
   toggled: boolean,
   onToggle: (e: {||}, toggled: boolean) => void,
   disabled?: boolean,
-  labelPosition: 'right',
+  labelPosition: 'right' | 'left',
 
   style?: {|
     marginTop?: number,
+    marginLeft?: number,
```

Comment:
Actually, I used this props to set marginLeft to 0

Reply:
I think you can set `marginLeft?: 0` in the typing to show that the only allowed value is 0 (useful to force future developers to avoid taking advantage of this - too much liberty is NOT something we want here).

Or as said in another comment, the component should apply no margin by itself - it's the parent that should in theory do the layouing with Column/ColumnStackLayout/LineStackLayout (or, in case they don't work, Spacer, but that's a band aid).